pkg/domain/infra/abi/play.go

```diff
@@ -942,5 +961,11 @@ func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, _ e
 	if err != nil {
 		return nil, err
 	}
+
+	// Stop and remove service container.
+	if _, err := ic.ContainerRm(ctx, []string{serviceContainerName(content)}, entities.RmOptions{Force: true}); err != nil {
```

Comment:
Should pod remove attempt to auto-remove service containers associated with the pod? Seems simpler

Reply:
Already doing that: see `vgrep maybeRemoveServiceContainer`

Reply:
Removed it. It's not necessary (anymore) here.